src/core/emulator.cpp

```diff
@@ -1624,6 +1624,16 @@ GraphicsSynthesizer& Emulator::get_gs()
     return gs;
 }
 
+EEBreakpointList* Emulator::get_ee_breakpoint_list()
```

Comment:
Recommend marking as `const`

```suggestion
EEBreakpointList* Emulator::get_ee_breakpoint_list() const
```

Reply:
sadly can't make this one const - it's returning a non-const pointer to a member